### ISSUE TITLE
[Bug] Does not compile on Zig 0.13.0

### ISSUE THREAD

**Issue by ZackeryRSmith** [KeithBrown39423/Hexdump#43]
The release version of Hexdump does not compile on 0.13.0 (or the master branch) of Zig.

**To Reproduce**

```bash
git clone https://github.com/KeithBrown39423/Hexdump
cd Hexdump
zig build
```

**Expected behavior**
A compiled binary of Hexdump using zig 0.13.0 (or master)

**Additional context**
This issue is not the cause of Hexdump but it's dependency, Clap. This issue is directly related to the `path` field in `LazyPath` being deprecated as of 0.13.0.

**Comment by ZackeryRSmith** [comment]
I would like to note Clap 0.9.1 will fix compiling on 0.12.0 **and** 0.13.0

The slight issue is the master branch isn't supported to any degree. Well I don't really see that as an issue.

**Comment by KeithBrown39423** [comment]
This is fixed in the dev branch, it just hasn't been merged into release yet.
